source/dynamical_systems/README.md

```diff
@@ -132,7 +132,7 @@ The following snippet illustrates the difference in these two options.
 ```c++
 // create a linear DS with attractor B in frame A
 state_representation::CartesianState BinA("B", "A");
-DynamicalSystems::Linear<state_representation::CartesianState> linearDS(BinA);
+dynamical_systems::Linear<state_representation::CartesianState> linearDS(BinA);
```

Comment:
In lines 68, 71, 93, 110, its still DynamicalSystem::Linear

Reply:
Ah I see, since that is the name of the base class I didn't change it. Thank you for catching it!
Clearly it was a typo originally, confusing the namespace with the class name. And that by itself is a great example why this refactor is a good idea!